### DISCUSSION TITLE
Configuration when using sway

### DISCUSSION THREAD

**Discussion by redat00** [ghostty-org/ghostty#4042] [Q&A] [answered]
Hello there !

I seem to have an issue, but I'm pretty sure it's more me that doesn't known how to configure Ghostty. 
Because a picture is better than words, here's the issue I'm facing. 

![image](https://github.com/user-attachments/assets/cfe36d7e-26f4-47e7-bf28-3ef04375a9b8)

As you can see when Ghostty is opening, it does not take the full height, neither the full width (when I'm opening it alone). I tried some configuration but I really don't know what settings I should use. On the left side I have WezTerm which does open to the full height and width.

If someone as any idea, I would love to have some insights ! 

Here's my current config (yes, there's nothing much for now) : 
```
gtk-titlebar = false
```

**Comment by redat00** [comment]
I'm starting to think that I'm the only one with this issue after seeing other running Sway or Hyprland (both based on Wayland) not having any problem at all... Not sure what I might be missing tho

**Comment by LionHeartP** [comment]
I'm using `window-decoration = false` in my config and it looks like this 
![image](https://github.com/user-attachments/assets/4b3f6ae3-ec19-4d00-9092-ac01621c3577)

Nvm. I just used `gtk-titlebar=false` and it also looks ok

**Reply by redat00** [reply]
Thanks for trying! Yeah I really don't know what's going on with my setup... I'm pretty sure my DE is telling Ghostty to behave like this, but I don't know what to do about it

**Comment by rhodes-b** [comment, marked as answer]
https://ghostty.org/docs/config/reference#fullscreen

or
https://ghostty.org/docs/config/reference#window-height
https://ghostty.org/docs/config/reference#window-height

**Reply by redat00** [reply]
Hi !

Fullscreen is not the behavior I'm looking for, as it is overtaking the whole screen... If I want a window on the left or the right of ghostty, I can't with this option... But it does "solve" the issue tho! 

And for the window-height and window-width.. I don't understand what values I am suppose to set... The documentation is unclear about it, I tried setting it to a very high value, as well as a small one, but it had no effect at all.. Do you have insights on what values might work ? 

Thanks for your answer !

**Comment by rhodes-b** [comment]
I believe I saw a similar post about someone with multiple monitor setup. Do you have 2 separately sized monitors?

There were quite a few people on the beta using sway (I just recently just switched to using it myself and see no issue)
